helm/hpcc/values.schema.json

```diff
@@ -156,6 +171,32 @@
       "required": [ "storageSize" ],
       "additionalProperties": false
     },
+    "resourceLimitItem": {
+      "type": "object",
+      "properties": {
+        "cpu": {
+          "oneOf": [
+            {
+              "type": "string",
+              "pattern": "^([+-]?[0-9.]+)([m]*)$"
+            },
+            { "type": "number" }
+          ]
+        },
+        "memory": {
+          "oneOf": [
+            {
+              "type": "string",
+              "pattern" : "^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$"
+            },
+            { "type": "integer" }
+          ]
+        }
+      }
+    },
+    "resources": {
+      "type": "object"
```

Comment:
resources object are no longer validated, in order to allow users to provide arbitary values for limits and requests.